packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -19,16 +20,23 @@ export const auth = (): Auth => {
     noAuthStatusMessage: authAuthHeaderMissing(),
   })(request);
 
-  const protect = createProtect({ request, authObject, notFound, redirect });
-  const redirectToSignIn = createRedirect({
-    redirectAdapter: redirect,
-    baseUrl: createClerkRequest(request).clerkUrl.toString(),
-    // TODO: Support runtime-value configuration of these options
-    // via setting and reading headers from clerkMiddleware
-    publishableKey: PUBLISHABLE_KEY,
-    signInUrl: SIGN_IN_URL,
-    signUpUrl: SIGN_UP_URL,
-  }).redirectToSignIn;
+  const clerkUrl = getAuthKeyFromRequest(request, 'ClerkUrl') as AuthStatus;
```

Comment:
☁️ `clerkUrl` is of type `AuthStatus` ? Is this some kind of typo? Also `getAuthKeyFromRequest` returns `string | null | undefined`, so the `clerkUrl` many not have a `.toString()` method.

Reply:
Good catch @dimkl 
TS did not complain as AuthStatus overlaps with string. Let me fix it!